README.md

```diff
@@ -27,15 +27,15 @@ below) and is faster, too.
 Create a runtime profile with
 
 ```
-python -mcProfile -o program.prof yourfile.py
+python -m cProfile -o program.prof yourfile.py
```

Comment:
A space after `-m` is common but not required:
```console
❯ python -mcProfile
Found existing alias for "python". You should use: "p"
Usage: cProfile.py [-o output_file_path] [-s sort] [-m module | scriptfile] [arg] ...

❯ python -m cProfile
Found existing alias for "python". You should use: "p"
Usage: cProfile.py [-o output_file_path] [-s sort] [-m module | scriptfile] [arg] ...
```

Reply:
```suggestion
python -m cProfile -o program.prof yourfile.py
```